CHANGELOG.md

```diff
@@ -3,6 +3,16 @@
 ## Upcoming
 
 * 3.0.next in progress
+  * Added in the ability to define the Environment on configuration of Sentry
+    (and not only per-event!)
+  * Added in the ability to disable the built-in UncaughtExceptionHandler in
+    order to allow defining our own.
+  * Added in a couple of examples:
+    * A basic example showing how to register a sentry-logger and how to fire
+      an event.
+    * An example that registers it's own uncaught exception handler to not
+      only log out to sentry, but also to log out to a pre-configured logger.
+  * Bumped `ring/ring-core` to `1.8.2`
```

Comment:
Curious the motivation for bumping here. Do we use stuff added in the new version?

Judy asking because we try to keep dependency requirements low not to force users to upgrade (and cause conflict with other dependencies)

Reply:
Hi, I normally strive to keep 3rd party dependencies up-to-date, esp when it brings in security updates. As an example, this ring core bump to 1.8.2 pulls in jetty-9.4.31.v2020072 which has security updates and other enhancements, as can be found [here](https://www.eclipse.org/lists/jetty-announce/msg00145.html).

Naturally, the user can pull in whichever Jetty they wish, or override dependencies as suits their needs.

Reply:
This makes sense when building applications but as a library author, this changes a bit.
The dependency here should be the lowest you need in order to build your library. Requiring anything higher will bring issues to the applications requiring your library. Since they have other dependencies, and some might have not yet updated and hence would not work together.

Reply:
Hi,

Sure, I agree - but given that 3.0.0 hasn't been released yet (and I've been using 1.8.2 of ring core in my other applications without issue), then we should be okay.

Rest assured, that the churn in Clojure (land) is dramatically less than the churn in other ecosystems, such as Javascript, which can change daily! :)

Reply:
Makes sense that you've been using 1.8.2 in your application. But it doesn't make sense to enforce this version on all library users. If you are using any API or feature from this version, it does not make any sense to bump it in a library. I'm really sorry but this is just guideline for library authoring, and we need to follow it here though.